client/client/app/components/ngbBlastSearchPanel/ngbBlastWholeGenomeView/ngbBlastWholeGenomeView.component.js

```diff
@@ -0,0 +1,11 @@
+import controller from './ngbBlastWholeGenomeView.controller';
+
+export default {
+    bindings: {
+        chromosomes: '<',
+        maxChrSize: '<',
```

Comment:
`maxChrSize` is redundant: we can compute this value based on `chromosomes` array *inside* `ngbBlastWholeGenomeView.component`